swagger-spec/users/addon_account_definition.yaml

```diff
@@ -0,0 +1,51 @@
+type: object
+title: External Account
```

Comment:
In order to reduce confusion (there's references to external account, addon, and third-party service when describing these endpoints), let's call this "Addon Account". 

I think in the endpoint descriptions (or in the addons section of the docs) we can further explain that addons _**are**_ third-party services such as GDrive, Dropbox, etc. 

Go ahead and remove all occurrences of the phrase "external account".